guides/telemetry.md

```diff
@@ -2,63 +2,67 @@ Amoc also exposes the following telemetry events:
 
 ## Scenario
 
-A telemetry span of a full scenario execution
+A telemetry span of a full scenario execution globally (i.e. the exported `init/0` function):
```

Comment:
Agree to everything except the idea of running `Scenario:start/` as a telemetry span in the `amoc_scenario` module, as this will run in the controller process and measure the time it takes to spawn an erlang process only, instead of measuring the lifespan of the entire amoc user. The current span must stay where it is already and it is the most useful one.

Reply:
I'm not sure what you mean here, I propose to measure exactly scenario callback time:
https://github.com/esl/amoc/blob/master/src/amoc_scenario.erl#L64

Reply:
well, all of the callbacks.

Reply:
and amoc_scenario module works as a synchronous wrapper for the callbacks. it doesn't spawn anything.